app/js/components/Wizard.jsx

```diff
@@ -22,11 +22,15 @@ class Wizard extends Component {
   canNext = () => {
     let currentStep = this.state.currentStep;
     const stepsLength = this.props.steps.length;
-    return currentStep < stepsLength - 1;
+    return currentStep <= stepsLength - 1;
   };
 
+  lastStep() {
+    return this.state.currentStep === this.props.steps.length - 1;
+  }
+
   next = () => {
-    if (!this.canNext()) {
+    if (!this.canNext() || this.lastStep()) {
```

Comment:
Not sure why you added `lastStep`. If in `canNext` you just remove the `=` you added, you don't need `lastStep()`

Reply:
Maybe you have a more elegant way of doing that:

* canNext is used to know when we display or not the next button. For the final steps we want to be able to display the final button so canNext must be true on last step
* next is used to perform the increase but in case of last step there is no increase

Reply:
Aaah, ok, so `canNext` is used somewhere else to display a button, but we stop it here by checking if it's the last and if so, we don't actually go next. Do I understand correctly?